zipline/gens/brokers/ib_broker.py

```diff
@@ -673,11 +694,16 @@ def order(self, asset, amount, style):
             order.m_orderType = "MKT"
         elif isinstance(style, LimitOrder):
             order.m_orderType = "LMT"
+            order.m_lmtPrice = limit_price
```

Comment:
I'm not sure if it is needed.  Please don't mix two different things in one PR.

Reply:
I don't know how to split a request. This change is needed as now you cannot post limit orders, it will fail on the absence of order.lmtPrice, is a bug

Reply:
I presume you mean 'PR' by request. 
You need to roll back this change, create a new branch, commit with a reasonable message and post a new pr.